### PR TITLE
Changelog for v46.17.0

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Changelog for v46.17.0

### CODE REVIEW

**Comment by brpowell** [comment]
@hbc-bindu Ready for review

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 46.17.0 - October 2, 2019
```
Comment (rcoringrato-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Shouldn't the date be: October 3?

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md]: 🤦‍♂

**Comment by AnanyaJha** [comment]
Will we need to update this again for the changes nate is adding?

**Comment by brpowell** [comment]
@AnanyaJha Yes, if it's merged I'll update it

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add `SFDX: Diff File Against Org` command to the context menu of metadata in the Explorer ([PR #1688](https://github.com/forcedotcom/salesforcedx-vscode/pull/1688), [Issue #1666](https://github.com/forcedotcom/salesforcedx-vscode/issues/1666))
+
+#### docs
+
+- Add documentation for LWC local development ([PR #1687](https://github.com/forcedotcom/salesforcedx-vscode/pull/1687))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: 46.16.0 - September 26, 2019 Changelog had an entry for LWC local development. Is this the same one or is this an update to that?

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md]: The new changes are adding/updating the doc

**Pull request merged**